changelog.txt

```diff
@@ -6,6 +6,11 @@ The **signac** package follows `semantic versioning <https://semver.org/>`_.
 
 Version 1
 =========
+next
+----
+
+- Fix issue get_project() "root" keyword failing silently
```

Comment:
Can you move this into the section under `[1.8.0]` / `Fixed` and include the issue / PR numbers? 1.8.0 hasn't been released yet, so that's the release that will contain this fix.